<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>410</width>
     <height>510</height>
    </size>
   </property>
   <widget class="QTabWidget" name="menu_tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="display_reminder_tab">
     <attribute name="title">
      <string>Display Reminders</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 9pt &quot;Calibri&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Reminders :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListView" name="reminderView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>321</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Century Gothic&quot;;
border-width : 1.2px;
border-color: lightblue;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancel_reminder_pushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>410</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
border-color: lightblue;
border-width : 1.2px;</string>
      </property>
      <property name="text">
       <string>Cancel Reminder</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="display_reminders_textEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="add_reminder_tab">
     <attribute name="title">
      <string>Add Reminder</string>
     </attribute>
     <widget class="QPushButton" name="add_reminder_pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;
border-color: lightblue;
border-width : 1.2px;</string>
      </property>
      <property name="text">
       <string>Add Reminder</string>
      </property>
     </widget>
     <widget class="QLabel" name="reminder_title_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Reminder Title&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="reminder_datetime_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Date &amp; Time</string>
      </property>
     </widget>
     <widget class="QLabel" name="reminder_timezone_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Time Zone&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="reminder_dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>290</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reminder_message_input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;
border-color: lightblue;
border-width : 1.2px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="reminder_message_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Reminder Message&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="reminder_title_input">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;
border-color: lightblue;
border-width : 1.2px;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="reminder_timezone_comboBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>261</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="add_reminder_textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>350</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Century Gothic'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>reminder_title_label</zorder>
     <zorder>reminder_datetime_label</zorder>
     <zorder>reminder_timezone_label</zorder>
     <zorder>reminder_dateTimeEdit</zorder>
     <zorder>reminder_message_input</zorder>
     <zorder>reminder_message_label</zorder>
     <zorder>reminder_title_input</zorder>
     <zorder>reminder_timezone_comboBox</zorder>
     <zorder>add_reminder_pushButton</zorder>
     <zorder>add_reminder_textEdit</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
